Glyssen/Dialogs/NarrationOptionsDlg.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Glyssen.Bundle;
+
+namespace Glyssen
+{
+	public partial class NarrationOptionsDlg : Form
+	{
+		private const int kInvalidNarratorNum = -1;
+		private int m_intNarratorNum = kInvalidNarratorNum;
+		private readonly Project m_project;
+
+		public NarrationOptionsDlg(Project project)
+		{
+			InitializeComponent();
+			m_project = project;
+
+			if (m_project.ProjectSettings.NumberOfNarrators != ProjectSettings.kNumberOfNarratorsNotSet)
+			{
+				m_intNarratorNum = m_project.ProjectSettings.NumberOfNarrators;
+				m_txtNarratorNum.Text = m_intNarratorNum.ToString();
+			}
+
+			switch (m_project.ProjectSettings.NarratorGenders)
+			{
+				case NarratorGenders.FemaleOnly:
+					m_rdoFemaleOnly.Checked = true;
+					break;
+				case NarratorGenders.MaleOrFemale:
+					m_rdoMaleOrFemale.Checked = true;
+					break;
+				default:
+					m_rdoMaleOnly.Checked = true;
+					break;
+
+			}
+		}
+
+		private bool m_validateAndSetNarratorNum()
```

Comment:
Most of this code can be eliminated by using a numeric up/down control instead of a text box. Note: we probably want to set a max value equal to the number of books in the project.